torchvision/io/_video_opt.py

```diff
@@ -392,6 +392,13 @@ def get_pts(time_base):
         audio_pts_range=audio_pts_range,
         audio_timebase=audio_timebase,
     )
+    _info = {}
```

Comment:
`_info` is filled in but is NOT returned from function. Why did you define it?

Reply:
Oups, this is a bug, I wanted to return `_info` instead, thanks for the catch!